src/bundle/streaming_writer.zig

```diff
@@ -9,30 +9,25 @@ const c = @cImport({
     @cInclude("zstd.h");
 });
 
+const Writer = std.io.Writer;
```

Comment:
There are a lot of Writers in zig, so I prefer the slightly longer explicit option here, but I'll modify that in the main PR.